README.md

```diff
@@ -134,6 +134,13 @@
 'GACCTCCTGCCTTCTCCTTCATT'
 ```
 
+### To RNA
```

Comment:
`#### To RNA`

Reply:
fixed [a8b5d17](https://github.com/openscilab/opr/pull/53/commits/a8b5d17507fa88ad6e1348746cb64003b2765cba).